packages/vc-verification/src/utils/errors.ts

```diff
@@ -25,11 +25,19 @@ export class CredentialRevoked extends Error {
 }
 
 export class RevokerNotAuthorized extends Error {
-  constructor(revoker: string, credential: string, reason?: string) {
+  constructor(
+    revoker: string,
+    credential: string,
+    reason?: string,
+    stack?: string
```

Comment:
@whitneypurdum Just curious, are we expecting some unexpected error?

Reply:
@nichonien this is to add a stack trace to see where ipfs is being read, or where the error is coming from. right ow we are just assuming where it is happening (in ew-did-registry). This will now provide the stack trace for the error so we can see where it is happening for sure.